### DISCUSSION TITLE
branched-chain polymer building

### DISCUSSION THREAD

**Discussion by hirose125** [marrink-lab/polyply_1.0#461] [Q&A]
Thank you for creating such a wonderful toolkit. Using this feature, is it possible to produce a branched-chain polymer with more than three reactive sites within the monomer?
![image](https://github.com/marrink-lab/polyply_1.0/assets/113399340/1846e3f2-afb2-453e-ac13-b60662d9e852)

**Comment by fgrunewald** [comment]
@hirose125 I'm happy you like the package. To answer your question, yes it is possible to implement a polymer as you have depicted. Depending on whether you want to use all-atom or coarse-grained parameters there are some optimizations that can be done, but I'm happy to help with that.

**Comment by hirose125** [comment]
@fgrunewald 
Thank you for your response.
I believe your tool is extremely useful for creating complex polymer simulation inputs. I'm planning to conduct all-atom MD simulations. If there are any examples, would it be possible to share them?

**Comment by fgrunewald** [comment]
@hirose125 I apologize for the late reply but I'm in the process of relocating and starting a new position. In order to prepare your simulations you first have to decide on a force-field. Ideally, the FF has a small molecule parameter generation program like CGENFF or LigParGen. You then want to obtain an itp file of the smallest fragment that contains all links possible. Subsequently, you'll need to extract the ff-files for which I and @ricalessandri can assist you with this.

**Comment by fgrunewald** [comment]
transferred to discussions
